### ISSUE TITLE
javascript minify not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Chris2011/js-css-minify-compress#11]
```
What steps will reproduce the problem?
1. Preferences -> Options -> JS CSS Minify Compress -> JavaScript Minify
2. Java Script Minify Settings -> uncheck Create New JS
3. No output

What is the expected output? What do you see instead?
Empty minified java script file

What version of the product are you using? On what operating system?
version 2.2.1


Please provide any additional information below.
It works fine before upgrade to version 2.2.1


```

Original issue reported on code.google.com by `yarow...@gmail.com` on 20 May 2015 at 6:46

**Comment by matthewslyman** [comment]
I experienced this issue, and managed to isolate the problem by process of elimination.
I had a variable (function parameter) called "char". Javascript ran fine, without errors; but "JS CSS Minify Compress" didn't like it: the software silently failed, created a blank output file (with just a newline char), and abandoned the minification process.

**Comment by Chris2011** [comment]
hey @matthewslyman is this the problematic code?

```javascript
var test = function(char) {
   console.log(char);
}
```

If not, please provide a sample code. Maybe it is fixed now with the new 2.3.0 version, which will be released in a couple of days.

**Comment by Chris2011** [comment]
Tested with the current 2.4.0 release and it is working as expected.
